common/components/stores/GroupSearchDispatcher.js

```diff
@@ -0,0 +1,10 @@
+// @flow
+
+'use strict';
```

Comment:
Is this required here? We don't traditionally declare `use strict` - though I don't see any harm if it is needed, of course.

Reply:
It was copied from the ProjectSearchDispatcher, which wasn't written by me.  Will go ahead and remove for consistency with the rest of our code.